File: nivel-02/ex-03/maior_que_10.js
var assert = require('assert');

var ehMaiorQue10 = function(valorQualquer) {
    if (valorQualquer > 10) {
        return 1;
    } else {
        return 0;
    }
}

try {
    assert.equal(1, ehMaiorQue10(17));
    assert.equal(0, ehMaiorQue10(9));
} catch(e) {
    console.log(e);
}

File: nivel-02/ex-03/maior_que_10.c
#include <stdio.h>
#include <assert.h>
#include <stdbool.h>

bool ehMaiorQue10(int valorQualquer) {
    if (valorQualquer > 10) {
        return true;
    } else {
        return false;
    }
}

int main (){

    assert(true == ehMaiorQue10(17));
    assert(false == ehMaiorQue10(9));

    return 0;
}